docs/architecture/adr-template.md

```diff
@@ -1,5 +1,11 @@
 # ADR {ADR-NUMBER}: {TITLE}
 
+## Status
+
+> A decision may be "proposed" if it hasn't been agreed upon yet, or "accepted" once it is agreed upon. Once the ADR has been implemented mark the ADR as "implemented". If a later ADR changes or reverses a decision, it may be marked as "deprecated" or "superseded" with a reference to its replacement.
+
+{Deprecated|Proposed|Accepted|Declined}
```

Comment:
Implemented is not here - should it be?

Reply:
It should be because the text above it:

> Once the ADR has been implemented mark the ADR as "implemented"

Reply:
Resolved by 7d564a1ff4c571744d63825869b246cc543ce01f